static/chrome/locale/en-US/wikicite.properties

```diff
@@ -67,7 +67,7 @@ wikicite.item-menu.get-indexer = Import from %1$s
 wikicite.item-menu.get-occ = Get citations from OpenCitations Corpus
 wikicite.item-menu.get-pdf = Extract from attachments
 wikicite.item-menu.import-citations = Import from file
-wikicite.item-menu.add-zotero = Add Zotero item(s)
+wikicite.item-menu.add-zotero = Add citations from Zotero item(s)
```

Comment:
I'm not sure this is right. This is shown when the user clicks on "+" in the Citations pane. Phrasing it this way implied (to me) that you will be importing citations from another Zotero item, when in reality you are adding zotero items _as_ citations.